packages/protocol/contracts/stability/Exchange.sol

```diff
@@ -113,11 +134,68 @@ contract Exchange is
     updateBucketsIfNecessary
     nonReentrant
     returns (uint256)
+  {
+    return _sell(sellAmount, minBuyAmount, sellGold);
+  }
+
+  /**
+   * @notice Exchanges sellAmount of sellToken in exchange for at least minBuyAmount of buyToken
```

Comment:
```
@notice Exchanges a specific amount of one token for an unspecified amount of another.
@param sellAmount The number of tokens to send to the exchange.
@param minBuyAmount The minimum number of tokens for the exchange to send in return.
@param sellGold True if the caller is sending CELO to the exchange, false otherwise.
@return The number of tokens sent by the exchange.
@dev The caller must first have approved `sellAmount` to the exchange.
```